ast/stdast.go

```diff
@@ -2768,6 +2768,8 @@ var p1 = &Source{
 		"\n",
 	},
 }
+
+// StdAst is the AST for the standard library.
```

Comment:
This file is autogenerated in `cmd/dumpstdlibast.go` and package `dump`. This comment needs to be generated, rather than inserted directly. An easy way to do it is to add `VariableConfig string`  to `dump.Options` and update `dump.dumpState.dump(...)` to use that comment.

Reply:
Instructions to regenerate this file may be handy: https://github.com/google/go-jsonnet#generated-stdlib

Reply:
Done